exercises/any_odds_and_evens/main_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import "testing"
+
+func TestAnyIsOdd(t *testing.T) {
+	for _, tc := range []struct {
+		s    []int
+		want bool
+	}{
+		{[]int{}, false},
```

Comment:
What about tests containing zeros? Consider adding them to other programs, where relevant.